components/ContentFlow.brs

```diff
@@ -221,7 +221,7 @@ sub onTruexAdDataReceived(event as object)
             vast_config_url: determineVastConfigUrl(decodedData.vast_config_url),
             placement_hash: decodedData.placement_hash
         },
-        supportsCancelStream: true, ' enables cancelStream event types, disable if Channel does not support
+        supportsUserCancelStream: true, ' enables cancelStream event types, disable if Channel does not support
```

Comment:
hey, the comment needs to be updated too -- `cancelStream` -> `userCancelStream`

Reply:
ok